buildtest/cli/__init__.py

```diff
@@ -1295,3 +1303,26 @@ def cdash_menu(subparsers):
     upload.add_argument(
         "-o", "--open", action="store_true", help="Open CDASH report in browser"
     )
+
+
+def help_all(subparsers):
+    """This method will add parser for hidden command that can be shown when using --help-all/-H
```

Comment:
please format `--help-all/-H` by surrounding with backticks "``". This will render properly in docstring